src/hooks/users/useRecentWorldUsers.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from "react";
+
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+import { normalizeTimestampToMilliseconds } from "utils/time";
+
+import { useUserLastSeenThreshold } from "hooks/useUserLastSeenThreshold";
+
+import { useWorldUsers } from "./useWorldUsers";
+
+export const useRecentWorldUsers = (): {
```

Comment:
[note] This hook was split out of `src/hooks/users/index.ts` into it's own file here.